### PR TITLE
chore: added types for element styles

### PR DESCRIPTION
**Pull request by anoushk1234**

### CODE REVIEW

File: packages/react-xnft/package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@coral-xyz/common-public": "*",
-    "@coral-xyz/themes": "*",
+    "@coral-xyz/common-public": "^0.2.0-latest.379",
```
Comment (hkirat) [inline packages/react-xnft/package.json:16]: Could you revert this

File: yarn.lock [hunk 2]
```diff
@@ -1634,7 +1634,24 @@
     stream-browserify "^3.0.0"
     util "^0.12.4"
 
-"@coral-xyz/xnft-cli@*", "@coral-xyz/xnft-cli@^0.0.1":
+"@coral-xyz/common-public@^0.2.0-latest.379":
```
Comment (hkirat) [inline yarn.lock:1637]: Could you revert this?

**Review by hkirat — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (anoushk1234) [reply yarn.lock:1637]: sure, although it wasn't allow me to build without these

**Comment by anoushk1234** [comment]
@hkirat is this gtg?

File: packages/react-xnft/package.json [hunk 3]
```diff
@@ -13,8 +13,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@coral-xyz/common-public": "*",
```
Comment (hkirat) [inline packages/react-xnft/package.json:16]: I think you ended up deleting these by mistake? Can you add those back

File: yarn.lock (hunk 2 above)
Comment (hkirat) [reply yarn.lock:1637]: I still see  the yarn changes, could you revert to the current lock file?

**Review by hkirat — CHANGES_REQUESTED** [review]

File: packages/react-xnft/src/reconciler.ts [hunk 4]
```diff
@@ -710,7 +710,23 @@ export const HOST: Host = {
   })(),
 };
 
-type Style = any;
+type _HostConfig = HostConfig<
```
Comment (hkirat) [inline packages/react-xnft/src/reconciler.ts:713]: I think this flowed over from a merge conflict, mind removing this as well? It's dead code

Comment (anoushk1234) [reply packages/react-xnft/src/reconciler.ts:713]: yea i think this is from your recent pr, i synced this before pushing and fixed the conflicts as well, ill remove it

**Pull request merged**